contracts/generate.go

```diff
@@ -17,6 +17,9 @@
 package contracts
 
 //go:generate abigen --sol ./bridge/Bridge.sol --pkg bridge --out ./bridge/Bridge.go
+//go:generate abigen --sol ./extbridge/ext_bridge.sol --pkg extbridge --out ./extbridge/ext_bridge.go
```

Comment:
Can i ask what is this for?

Reply:
This code make us to generate go binding code of the solidity file like below.

```
➜  klaytn git:(190723e_refactor_contract_method_parameter) cd contracts 
➜  contracts git:(190723e_refactor_contract_method_parameter) go generate
```

And then you can find the genearted file like below.
`./extbridge/ext_bridge.go`